src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -549,6 +554,89 @@ private JsValue Regex(JsValue self, JsValue[] args)
                 return new JsValue(regex.IsMatch(args[0].AsString()));
             }
 
+            private static JsValue ScalarToRawString(JsValue self2, JsValue[] args)
+            {
+                if (args.Length != 2)                
+                    throw new InvalidOperationException("scalarToRawString(document, lambdaToField) may be called on with two parameters only");
+
+
+                JsValue firstParam = args[0];
+                if (firstParam.IsObject() == false || args[0].AsObject() is BlittableObjectInstance == false)
+                {
+                    throw new InvalidOperationException("scalarToRawString(document, lambdaToField) may be called with a document first parameter only");
+                }
+
+                JsValue secondParam = args[1];
+                if (args[1].IsObject() == false || secondParam.AsObject() is FunctionInstance == false)
+                    throw new InvalidOperationException("scalarToRawString(document, lambdaToField) must be called with a second lambda argument");
+
+
+
+                BlittableObjectInstance selfInstance = firstParam.AsObject() as BlittableObjectInstance;
+                FunctionInstance lambda = secondParam.AsObject() as FunctionInstance;
```

Comment:
Replace all this with TryGetField from 4.1 branch

Reply:
See: https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Server/Documents/Indexes/Static/JavaScriptIndex.cs#L479-L492

Reply:
ok